cores/nRF5/freertos/config/FreeRTOSConfig.h

```diff
@@ -196,8 +196,8 @@ standard names - or at least those used in the unmodified vector table. */
      */
 #define configUSE_DISABLE_TICK_AUTO_CORRECTION_DEBUG     0
 
-// Sysview require at least debug level 3
-#if CFG_DEBUG >= 3
+// Sysview is enabled by default at debug level 3
```

Comment:
the comment text is not relevant anymore, you can safely remove it

Reply:
<details><summary>Same as prior response...</summary><P>

[Prior response](https://github.com/adafruit/Adafruit_nRF52_Arduino/pull/469#discussion_r397044991) said the following:

> I think this comment is very accurate. When the debug level is set to 3, SysView is enabled. This is the old behavior, and continues to be the behavior with this PR. So, I am not sure I understand why this would not be a relevant comment?

</details>

Reply:
@henrygab yeah, previously the text is meant to explain why we enable sysview when DEBUG >= 3, now sysvew has its own macro which is self described already. So it is safe to remove, though it is not really an important at all.